app/views/shared/_back_to_top_link.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= link_to(
+  target_id,
+  class: "govuk-link govuk-link--no-visited-state back-to-top-link"
+) do %>
+  <svg role="presentation" focusable="false" class="back-to-top-icon" xmlns="http://www.w3.org/2000/svg" width="13" height="17" viewBox="0 0 13 17">
+    <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
+  </svg>
+  <p><%= t(".back_to_top") %></p>
+<% end %>
```

Comment:
There's currently no 'Back to top' link in the gov UK design system, although there's one [used on their site](https://design-system.service.gov.uk/get-started/), and [an open issue to discuss it](https://github.com/alphagov/govuk-design-system-backlog/issues/5). 

I've put this in a shared partial so we can use it elsewhere as needed until, hopefully, an official one is added!